diff --git rules.bzl rules.bzl
index e310c2a..662386b 100644
--- rules.bzl
+++ rules.bzl
@@ -25,8 +25,9 @@ def _android_ndk_repository_impl(ctx):
     """
     ndk_path = ctx.attr.path or ctx.os.environ.get("ANDROID_NDK_HOME", None)
     if not ndk_path:
-        fail("Either the ANDROID_NDK_HOME environment variable or the " +
-             "path attribute of android_ndk_repository must be set.")
+        # Create an empty repository that allows non-Android code to build.
+        ctx.template("BUILD.bazel", ctx.attr._template_empty)
+        return None
 
     if ctx.os.name == "linux":
         clang_directory = "toolchains/llvm/prebuilt/linux-x86_64"
@@ -121,7 +122,9 @@ android_ndk_repository = repository_rule(
         "_template_target_systems": attr.label(default = ":target_systems.bzl.tpl", allow_single_file = True),
         "_template_ndk_clang": attr.label(default = ":BUILD.ndk_clang.tpl", allow_single_file = True),
         "_template_ndk_sysroot": attr.label(default = ":BUILD.ndk_sysroot.tpl", allow_single_file = True),
+        "_template_empty": attr.label(default = ":BUILD.empty.tpl", allow_single_file = True),
     },
+    environ = ["ANDROID_NDK_HOME"],
     local = True,
     implementation = _android_ndk_repository_impl,
 )
