diff --git MODULE.bazel MODULE.bazel
index a7cf644..e4007df 100644
--- MODULE.bazel
+++ MODULE.bazel
@@ -10,6 +10,7 @@ bazel_dep(name = "bazel_skylib", version = "1.6.1")
 bazel_dep(name = "rules_java", version = "7.2.0")
 bazel_dep(name = "rules_python", version = "0.23.1")
 bazel_dep(name = "rules_cc", version = "0.0.8")
+bazel_dep(name = "rules_android", version = "2.0.0")
 
 rules_kotlin_extensions = use_extension("//src/main/starlark/core/repositories:bzlmod_setup.bzl", "rules_kotlin_extensions")
 use_repo(
@@ -19,7 +20,6 @@ use_repo(
     "com_github_jetbrains_kotlin",
     "com_github_pinterest_ktlint",
     "released_rules_kotlin",
-    "rules_android",
 )
 
 # Once the released rules_koltin is defined, configure it.
diff --git third_party/BUILD third_party/BUILD
index 624fe5f..0e403f4 100644
--- third_party/BUILD
+++ third_party/BUILD
@@ -76,7 +76,7 @@ java_binary(
 # TODO(bazelbuild/rules_kotlin/issues/273): Remove android_sdk import.
 java_import(
     name = "android_sdk",
-    jars = ["@bazel_tools//tools/android:android_jar"],
+    jars = ["@rules_android//tools/android:android_jar"],
     neverlink = 1,
     visibility = ["//visibility:public"],
 )
